### DISCUSSION TITLE
Data Explorer: One search field to query over all columns (just like in RStudio)

### DISCUSSION THREAD

**Discussion by cutterkom** [posit-dev/positron#4746] [Feedback] [answered]
I am really impressed by Positron! I love the combination of RStudio and VS Code. (Split screen!)

But there is one thing that make my data exploration way slower than in R Studio and is a dealbreaker for my, unfortunately. 

The Position Data Explorer makes querying way slower. RStudio has one search field and it searches over all columns. I need that as well in Positron, the current way is way to much clicking. 

Thanks for your work!

**Comment by petetronic** [comment, marked as answer]
Thank you @cutterkom, your feedback is valuable to hear! We're tracking this in https://github.com/posit-dev/positron/issues/3302 but please add a note there as that will help increase our signal on what matters most to users.

**Reply by jthomasmock** [reply]
Thanks for the feedback @cutterkom ! I've added it to the 3302 issue for you.
